docs/index.yml

```
@@ -1,111 +1,132 @@
-### YamlMime:Landing
+### YamlMime:Hub
 
 title: Office Add-ins documentation # < 60 chars
 summary: Use the Office Add-ins platform to build solutions that extend Office applications and interact with content in Office documents and in Outlook mail messages and calendar items. With Office Add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build solutions that can run in Office on the web, Windows, Mac, Android, and iOS. # < 160 chars
+brand: office
 
 metadata:
   title: Office Add-ins documentation # Required; page title displayed in search results. Include the brand. < 60 chars.
   description: Learn how to build Office Add-ins. # Required; article description that is displayed in search results. < 160 chars.
-  ms.prod: non-product-specific #Required
-  ms.topic: landing-page # Required
+  services: service 
+  ms.service: service
+  ms.topic: hub-page # Required
   author: o365devx #Required; your GitHub user alias, with correct capitalization.
   ms.author: o365devx #Required; microsoft alias of author; optional team alias.
-  ms.date: 07/31/2020 #Required; mm/dd/yyyy format.
+  ms.date: 02/03/2021 #Required; mm/dd/yyyy format.
 
-# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
+# highlightedContent section
+# Maximum of 8 items
+highlightedContent:
+  items:
+    # Card 1
+    - itemType: overview
+      title: Office Add-ins platform overview
+      url: overview/office-add-ins.md
+    # Card 2
+    - itemType: learn
+      title: "MS Learn: Introduction to Office client customization with add-ins"
+      url: https://docs.microsoft.com/learn/modules/intro-office-add-ins/
+    # Card 3
+    - itemType: learn
+      title: "MS Learn: Extend Office clients with Office add-ins"
+      url: https://docs.microsoft.com/learn/paths/m365-office-add-in-associate/
+    # Card 4
+    - itemType: whats-new
+      title: Microsoft 365 Developer Blog
+      url: https://developer.microsoft.com/office/blogs/technology/add-ins/
+    # Card 5
+    - itemType: how-to-guide
+      title: Beginner's guide
+      url: overview/learning-path-beginner.md
+    # Card 6
+    - itemType: reference
+      title: API reference documentation
+      url: https://docs.microsoft.com/javascript/api/overview
+    # Card 7
+    - itemType: sample
+      title: Patterns and Practices GitHub repository
```

Could we brand this like the MS Learn links? Something about "repository" feels too much like an implementation detail, rather than something that'll guide users.
```suggestion
      title: GitHub: Patterns and Practices for Office Add-ins
```